custom/inddex/food.py

```diff
@@ -291,7 +291,9 @@ def _set_conversion_factors(self):
     @property
     def age_range(self):
         if not self.age_months_calculated:
-            return None
+            return "unknown"
+        if not self.age_years_calculated:
```

Comment:
I think this one should be of the form:
```suggestion
        if self.age_years_calculated is None:
```
I assume anyone under 1 yr will have `age_years_calculated` as `0`, which is falsy.  Probably safest to change the months calculation as well, in case any < 1 m/o babies are in the system.

This might also be good to add to the test dataset.

Reply:
Thanks, Ethan, I have updated.

Re. the tests, I have spent about half hour trying to figure out where this part is getting tested and couldn't find. There is the test data called `data_source.csv`, but if I had one row there I wasn't able to figure out what all other reports will have the data updated. It would have been very easy if the tests were testing individual units rather than test the whole piece in one go.

Reply:
Yeah, there is a lot of setup necessary for anything to run, since the reports rely on a UCR based on parent and child cases and a bunch of types of fixtures.  The master data set has several hundred indicators, so writing an individual test for each would be challenging, especially since most of them are either pulled directly from the UCR or generated programmatically.  For the most part, the complexity is in the glue code linking all that stuff together, not so much the individual indicators, so that's primarily what I tested.  This approach also means a non-developer can modify or verify test cases and output without needing to understand how the code works (that's how I got the initial test cases).

I should've put in a README explaining where this stuff is, but here's a quick overview:

The code includes realistic example data in the `example_data` directory.  This is used by the `populate_inddex_test_domain` management command to bootstrap a local environment.  This same dataset is also used for tests.

The case data is in `foodrecall_cases.csv` and `food_cases.csv` (the former is where `age_years_calculated` is initially supplied).  The master report is based primarily on the `food_consumption_indicators.json` data source, which has a row for each food case, but includes information from the parent case.  The master report also relies on the fixtures set up by `example_data`.  When updating the reports or the base data set, you can update the expected reports directly with how the output should change.  Alternatively, once you're sure the output is correct, you can also temporarily uncomment the `_overwrite_report` lines in the tests to write the output to disk.

Reply:
Ah, that makes sense! It's indeed very challenging to test custom reports. I have added this note to the README

Reply:
great, thanks!